### PR TITLE
fix: Consistent model selection in parsing

### LINKED ISSUE #140: Schema option case sensitivity
Docs say you can use:

```
    dbt_schema_excludes:
      - development
      - testing
```

And there's also support for a `--dbt_schema_excludes` command-line option.

However, you must actually upcase the schema name, unlike the example before, because the code that actually looks at this is:

```
if schema_excludes and node["schema"].upper() in schema_excludes:
```

I'm not really clear why the `upper()` is there. If it's just for the sake of normalising, then we also need to normalise the arguments?

### PR DESCRIPTION
**Pull request by gouline**
Resolves #140

### CODE REVIEW

**Comment by gouline** [comment]
@willbryant See if this works for you before I merge and release

**Comment by willbryant** [comment]
@gouline Nope it's actually doing completely NOOP runs on this branch. No further output after the "Session established successfully" banner.

Rolled back to 0.9.7 and it's working again, so don't think it's something wrong with my configuration.

**Comment by gouline** [comment]
Drop any details of your configuration that I would need to reproduce and I'll look into it when I have time.

**Comment by gouline** [comment]
Actually @willbryant give it another go, I fixed a silly typo that may have been causing the noop 🤔

**Comment by willbryant** [comment]
It's still skipping for me. `-v` shows:

```
DEBUG    Skipping DIM_CUSTOMERS in schema BI not in target schema PUBLIC                                                                                                                                                                                    dbt_manifest.py:65
```

Which is not what the previous version did. It starts working again if I add:

```
dbt_schema: bi
```

To my config. Whereas before, I just had:

```
  dbt_schema_excludes:
  - bi_dbt_ga4
```

IOW I think the behavior change is that now effectively only one schema at a time is supported?

File: dbtmetabase/parsers/dbt_manifest.py [hunk 1]
```diff
@@ -111,46 +97,41 @@ def read_models(
...
-                logger().debug("Skipping %s not of resource type source", model_name)
+            if source_database != self.database:
+                logger().debug(
+                    "Skipping %s not in target database %s", source_name, self.database
+                )
                 continue
 
-            if schema and node["schema"].upper() != schema.upper():
-                # Skip any models not in target schema
+            if self.schema and source_schema != self.schema:
```
Comment (willbryant) [inline dbtmetabase/parsers/dbt_manifest.py:114]: `self.schema` will always be set now I think?

**Comment by gouline** [comment]
That makes sense, I was testing with the folder parser. Latest commit should fix the issue you raised.

**Comment by willbryant** [comment]
Yup that's working - thanks.

**Pull request merged**